src/pfs/pfs_test.go

```diff
@@ -10,6 +10,13 @@ import (
 	"google.golang.org/protobuf/testing/protocmp"
 )
 
+func TestCommit_NilBranch(t *testing.T) {
```

Comment:
Can we also add a test where `Branch.name` originally has a non-empty value?

Reply:
I'll fix that.